rambo/options.py

```diff
@@ -2,6 +2,19 @@
 from rambo.settings import SETTINGS, PROJECT_LOCATION, PROJECT_NAME
 from rambo.utils import get_env_var, set_env_var
 
+def guest_name_option(guest_name=None):
+    '''Validate guest_name. If not supplied, set to default. Set as env var.
+
+    Args:
+        guest_name (str): Guest name to use.
+
+    Return guest_name (str)
+    '''
+    if guest_name:
+        return guest_name
+    else:
+        return "DEFAULT"
```

Comment:
This is where the env var should be set, like the other options in this file and as indicated by the docstring. Also I'm not sure what validation to do here, but none is being done currently. What validation is done might change whether this is for setting the VM's name, or the host name. I'm not sure off the top of my head.

I'd also suggest the pattern in the other functions, e.g. from `def guest_os_option`:
```python
if not guest_os:                                                                                                                                          
    guest_os = SETTINGS['GUEST_OSES_DEFAULT']

# Input validation

set_env_var('guest_os', guest_os)
return guest_os
```
(with the relevant substitutions of course)

Reply:
This placement of `set_env_var` is different actually, because it comes after the validation here. I think I like it better because you can autocorrect in the validation section before setting the env var, also it's conceptually similar to a return statement ("I'm done with it, here it is.")

Here or later we should move the others to conform :)